src/drawing.c

```diff
@@ -86,7 +87,7 @@ void tfb_draw_hline(int x, int y, int len, u32 color)
       return;
 
    len = MIN(len, MAX(0, (int)__fb_win_end_x - x));
-   memset32(__fb_buffer + y * __fb_pitch + (x << 2), color, len);
+   memset_var(__fb_buffer + y * __fb_pitch + (x << (__fb_depth >> 4)), color, len, __fb_depth);
```

Comment:
NIT: coding style, line longer than 80 cols. Can you break it like:

```suggestion
   memset_var(__fb_buffer + y * __fb_pitch + (x << (__fb_depth >> 4)),
              color, len, __fb_depth);
```